src/statsbomb_snowflake/loader.py

```
@@ -0,0 +1,361 @@
+            return len(matches)
+
+        loaded_at = self._get_load_timestamp()
+        rows_loaded = 0
+
+        with self.cursor(conn) as cur:
+            # Delete existing matches for this competition/season (incremental load)
+            cur.execute(
+                f"""
+                DELETE FROM {table}
+                WHERE competition_id = %s AND season_id = %s
+                """,
```

Ruff (0.14.14)

Possible SQL injection vector through string-based query construction

(S608)